plugins/nirmata/api_token.go

```diff
@@ -0,0 +1,89 @@
+package nirmata
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://downloads.nirmata.io/nctl/downloads/"),
+		ManagementURL: sdk.URL("https://www.nirmata.io/security/login.html"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Nirmata.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 116,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.Email,
+				MarkdownDescription: "Email address registered in Nirmata.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Address,
+				MarkdownDescription: "Url address of Nirmata[https://nirmata.io].",
+				Optional:            true,
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+
```

Comment:
This empty line is not needed. Could we please make sure to fix all empty lines and spaces?